frontend/src/constants/paths.tsx

```diff
@@ -0,0 +1,7 @@
+export  const PATH_FOR_START_VIEW : string = "/";
```

Comment:
Jeśli to będzie w wielu miejscach, to spoko, że jest do osobnego pliku. Na razie zostawmy. Ale jeśli będą w jednym to bym to inlinował. Bo wtedy od razu też widać wartość.